sound/hybrid_font.h

```diff
@@ -120,6 +121,36 @@ class HybridFont : public SaberBase {
       PlayPolyphonic(effect);
     }
   }
+  
+  void CommonSwing(Effect* monophonic, Effect* polyphonic) override {
```

Comment:
Why pass in the effect here?
The hybrid font already knows what effect to use.
Also, I think "StartSwing()" would be a better name.

Reply:
did this so I could assign a polyphonic_player object and manipulate volume.
I'll see if I can do that without passing the effects and change the function name

Reply:
I think I need to pass the effects in, but I'll change the function name.